plugin/storage/badger/dependencystore/storage.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dependencystore
+
+import (
+	"time"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+const (
+	dependencyKeyPrefix byte = 0xC0 // Dependency PKs have first two bits set to 1
+)
+
+// DependencyStore handles all queries and insertions to Cassandra dependencies
+type DependencyStore struct {
+	// session                  cassandra.Session
+	// dependencyDataFrequency  time.Duration
+	// dependenciesTableMetrics *casMetrics.Table
+	// logger                   *zap.Logger
```

Comment:
what is the plan with the commented out code?

Reply:
Forgotten comments only, the original wish was not to include dependencystore in the first PR. Is this still the case?